source/qcommon/steam.c

```diff
@@ -51,16 +53,18 @@ static const STEAMSHIM_Event* blockOnEvent(STEAMSHIM_EventType type){
 		}
 	}
 }
-
+cvar_t *steam_debug;
 /*
 * Steam_Init
 */
 void Steam_Init( void )
 {
+	 steam_debug = Cvar_Get( "steam_debug", "0", 0);
+
 #if DEDICATED_ONLY 
-	int r = STEAMSHIM_init(false, true);
+	int r = STEAMSHIM_init(steam_debug->integer, false, true);
```

Comment:
`false, true, false, true`
What is what?
I'd suggest using flags or a struct with options or the like.